TriangleCheckerTest.java

```diff
@@ -11,12 +11,22 @@ public void testValidTriangle() {
         float b = 3;
         float c = 4;
 
-        // Act
-        TriangleChecker.TriangleType type = TriangleChecker.checkTriangle(a, b, c);
-
         // Assert
-        assertEquals(TriangleChecker.TriangleType.NORMAL, type);
+        
+        //normal
+        assertEquals(TriangleChecker.TriangleType.NORMAL, TriangleChecker.checkTriangle(a, b, c));
+        //gleichseitig
+        assertEquals(TriangleChecker.TriangleType.EQUILATERAL, TriangleChecker.checkTriangle(2, 2, 2));
 
-    }
+        //kein Dreieck
+          assertEquals(TriangleChecker.TriangleType.NONE, TriangleChecker.checkTriangle(-1, 1, 1));
+          assertEquals(TriangleChecker.TriangleType.NONE, TriangleChecker.checkTriangle(1, -1, 1));
+          assertEquals(TriangleChecker.TriangleType.NONE, TriangleChecker.checkTriangle(1, 1, -1));
+          assertEquals(TriangleChecker.TriangleType.NONE, TriangleChecker.checkTriangle(5, 1, 1));
+          assertEquals(TriangleChecker.TriangleType.NONE, TriangleChecker.checkTriangle(1, 5, 1));
+          assertEquals(TriangleChecker.TriangleType.NONE, TriangleChecker.checkTriangle(1, 1, 5));
 
+        //gleichschenkliges Dreieck
+        assertEquals(TriangleChecker.TriangleType.ISOSCELES, TriangleChecker.checkTriangle(1, 2, 1));
```

Comment:
Die Testfälle sollten dringend in mehrere Testmethoden aufgeteilt werden, die sprechende Namen besitzen, wasd getestet wird

Reply:
Erledigt :)